### PR TITLE
Rollbar documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for Rollbar integration.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by RohitKhatta — APPROVED** [review]

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Rollbar**.
+2. Assign a name to the destination and click on **Continue**.
+
+## Connection settings
+
+To successfully configure Rollbar as a destination, you will need to configure the following settings:
+
+- **Access Token**: Enter your Rollbar access token. For more information on obtaining your Rollbar access token, refer to the <Link to="#faq">FAQ</Link> section below.
+- **Capture Uncaught Exceptions**: This setting lets you record any uncaught exceptions from the `window.oneerror` toggle.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/rollbar.mdx:34]:
```suggestion
- **Capture Uncaught Exceptions**: This setting is enabled by default and lets you record any uncaught exceptions from the `window.oneerror` toggle.
```

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- **Capture Unhandled Rejections**: Enable this setting to capture and report any unhandled Promise rejections, for example, Promise failures.
+- **Guess Uncaught Frames**: When this option is enabled, Rollbar guesses the frames in which an error was thrown. This is helpful in cases where the browser does not provide any line or column numbers.
+- **Code Version**: Enter the version string, for example, a version number. This setting is applicable if you have enabled the **Source Maps** setting in Rollbar.
+
+<div class="infoBlock">
+Refer to the <a href="https://docs.rollbar.com/docs/source-maps">Rollbar source maps documentation</a> for more information.
+</div>
+
+- **Environment**: Specify the environment in which your code is running. If you do not set any environment name, the events will be visible under the **unknown** environment in Rollbar.
+- **Ignored Messages**: Rollbar will ignore the excepts matching the exception messages added in this field. For example, you can ignore an exception message for a third-party browser plugin that is throwing errors.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/rollbar.mdx:44]:
```suggestion
- **Ignored Messages**: Rollbar will ignore the messages added in this field. For example, you can ignore an exception message for a third-party browser plugin that is throwing errors.
```
If it is technically correct

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- **Source Map Enabled**: Enable this setting for Rollbar to perform efficient stack tracing and error grouping.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/rollbar.mdx:45]:
```suggestion
- **Source Map**: Enable this setting for Rollbar to perform efficient stack tracing and error grouping.
```

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Rollbar.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/rollbar.mdx:46]:
```suggestion
- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Rollbar. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
```

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+<div class="infoBlock">
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/rollbar.mdx:48]: can remove

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/streaming-destinations/rollbar.mdx (hunk 5 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rollbar.mdx:48]:
```suggestion
```

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rollbar.mdx:49]:
```suggestion
```

File: docs/destinations/streaming-destinations/rollbar.mdx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+</div>
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/rollbar.mdx:50]:
```suggestion
```

**Pull request merged**